apps/sms/views/inbox/js/inbox.js

```diff
@@ -2,9 +2,9 @@
 /* vim: set shiftwidth=2 tabstop=2 autoindent cindent expandtab: */
 
 /*global Template, Utils, Threads, Contacts, Threads,
-         WaitingScreen, MessageManager, TimeHeaders,
```

Comment:
I removed all the TimeHeader in the inbox, but I think we still need to update the time string in subheader in the end. Not sure if it's a good idea to set the list data to timeheader initially?